src/autobot.ts

```diff
@@ -43,22 +39,6 @@ export interface Hooks {
   [ExecutionScope.PullRequest]: {
     /** Allows a plugin to modify the config before it's used anywhere */
     modifyConfig: AsyncSeriesWaterfallHook<[Config]>;
-    /**
-     * If a plugin tapped into this method returns `true`, processing of the current
-     * PR is ended early for all plugins. The plugin can then use `modifySkipStatus`
-     * to update the GitHub status of the PR.
-     */
-    shouldSkipAllProcessing: AsyncParallelBailHook<[PRContext, Config], void | true>;
-    /**
-     * When processing is skipped, plugins have a change to alter the status that will
-     * be reported on the PR. The default status state is `success` with no description.
-     */
-    modifySkipStatus: AsyncSeriesWaterfallHook<[Status, PRContext, Config]>;
-    /**
-     * Gives plugins an opportunity to do whatever it may need to do when PR processing
-     * is skipped. This is called _after_ the skip status has been set.
-     */
-    onSkip: AsyncParallelHook<[PRContext, Config]>;
```

Comment:
I found that this whole setup was unnecessary and was just created more issues than it was worth. Giving any random plugin the ability to pump the breaks on the whole process was a little... short sighted.